requirements/testing/minver.txt

```diff
@@ -10,3 +10,4 @@ pillow==8.0.0
 pyparsing==2.3.1
 python-dateutil==2.7
 fonttools==4.22.0
+pytest==3.6.0
```

Comment:
It appears you have accidentally included your possible change for #23154 here. Please put that in a separate PR. Note that it is best to use a branch other than `main` in order to avoid conflating PRs like this.
https://matplotlib.org/devdocs/devel/development_workflow.html

Reply:
So, I should put the pull request in a separate branch in the original scipy repo itself right?

Reply:
And should I create a new branch in scipy or should I do it on an existing branch?

Reply:
@harshvardhan2707 we have some guidance on working with branches here:
https://matplotlib.org/devdocs/devel/development_workflow.html